packages/wrangler/src/deployments.ts

```diff
@@ -40,8 +41,11 @@ export async function deployments(
 	);
 
 	const scriptTag = scriptMetadata.default_environment.script.tag;
+	const params = new URLSearchParams({ order: "asc" });
 	const { items: deploys } = await fetchResult<DeploymentListRes>(
-		`/accounts/${accountId}/workers/versions/by-script/${scriptTag}`
+		`/accounts/${accountId}/workers/deployments/by-script/${scriptTag}`,
+		undefined,
```

Comment:
What is this `undefined` (just curious)?

Reply:
``` javascript
export async function fetchResult<ResponseType>(
	resource: string,
	init: RequestInit = {},
	queryParams?: URLSearchParams,
	abortSignal?: AbortSignal
): Promise<ResponseType>
```
& is the function signature for `fetchResult`. You can pass `init` for your request content but I'm not doing that so leaving it as `undefined`.
There might be a way to not specify it all using named parameters but I couldn't see how to do that

Reply:
Yeah this is JavaScript idiosyncratic behavior for Web APIs lol